sdk/eventgrid/Azure.Messaging.EventGrid.SystemEvents/README.md

```diff
@@ -21,38 +21,89 @@ dotnet add package Azure.Messaging.EventGrid.SystemEvents --prerelease
 
-### Authenticate the client
+### Receiving and Deserializing Events
+There are several different Azure services that act as [event handlers](https://docs.microsoft.com/azure/event-grid/event-handlers).
 
-In order for the client library to interact with a topic or domain, you will need the `endpoint` of the Event Grid topic and a `credential`, which can be created using the topic's access key.
+Note: if using Webhooks for event delivery of the *Event Grid schema*, Event Grid requires you to prove ownership of your Webhook endpoint before it starts delivering events to that endpoint. At the time of event subscription creation, Event Grid sends a subscription validation event to your endpoint, as seen below. Learn more about completing the handshake here: [Webhook event delivery](https://docs.microsoft.com/azure/event-grid/webhook-event-delivery). For the *CloudEvents schema*, the service validates the connection using the HTTP options method. Learn more here: [CloudEvents validation](https://github.com/cloudevents/spec/blob/v1.0/http-webhook.md#4-abuse-protection).
 
-You can find the endpoint for your Event Grid topic either in the [Azure Portal](https://portal.azure.com/) or by using the [Azure CLI](https://docs.microsoft.com/cli/azure) snippet below.
+Once events are delivered to the event handler, we can deserialize the JSON payload into a list of events.
 
-```bash
-az eventgrid topic show --name <your-resource-name> --resource-group <your-resource-group-name> --query "endpoint"
+```C# Snippet:SystemEventsCloudEventParseJson
+var bytes = await httpContent.ReadAsByteArrayAsync();
+// Parse the JSON payload into a list of events
+CloudEvent[] cloudEvents = CloudEvent.ParseMany(new BinaryData(bytes));
 ```
-
-The access key can also be found through the [portal](https://docs.microsoft.com/azure/event-grid/get-access-keys), or by using the Azure CLI snippet below:
-```bash
-az eventgrid topic key list --name <your-resource-name> --resource-group <your-resource-group-name> --query "key1"
-```
-
-#### Authenticate using Topic Access Key
-
-Once you have your access key and topic endpoint, you can create the publisher client as follows:
-```C# Snippet:CreateNamespaceClient
-// Construct the client using an Endpoint for a namespace as well as the shared access key
-var senderClient = new EventGridSenderClient(new Uri(namespaceTopicHost), topicName, new AzureKeyCredential(namespaceKey));
+#### Deserializing event data
+From here, one can access the event data by deserializing to a specific type by calling `ToObjectFromJson<T>()` on the `Data` property. In order to deserialize to the correct type, the `EventType` property (`Type` for CloudEvents) helps distinguish between different events. Custom event data should be deserialized using the generic method `ToObjectFromJson<T>()`. There is also an extension method `ToObject<T>()` that accepts a custom `ObjectSerializer` to deserialize the event data.
```

Comment:
```suggestion

From here, one can access the event data by deserializing to a specific type by calling `ToObjectFromJson<T>()` on the `Data` property. In order to deserialize to the correct type, the `EventType` property (`Type` for CloudEvents) helps distinguish between different events. Custom event data should be deserialized using the generic method `ToObjectFromJson<T>()`. There is also an extension method `ToObject<T>()` that accepts a custom `ObjectSerializer` to deserialize the event data.
```

Reply:
FYI I don't think this impacts the rendered markdown